pkg/rest/request.go

```diff
@@ -50,3 +51,18 @@ func Post(url string, body, responseModel any) error {
 
 	return nil
 }
+
+// Get sends a GET request to the given URL and returns the response body as a byte slice.
+func Get(url string) ([]byte, error) {
```

Comment:
This is somewhat of an anti pattern but ssince we're going to move away from this repo or refactor over time I'll ignore it but generally, unless there are enrichments to the request, there's no need to wrap the call. Also, it's perferred to use a request and set a timeout as this could spin indefinitely otherwise.

Reply:
You are totally right, fixing it.